packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -21,7 +21,8 @@
   flex-grow: 1;
   width: var(--width-left-navigation);
   z-index: 19999;
-  background: var(--color-accent);
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
```

Comment:
Perhaps we can update the [stylelint config](https://github.com/commercetools/merchant-center-application-kit/blob/main/stylelint.config.js#L39) to include the tokens from the `theming.ts` file? (although we would need to read a typescript file from a javascript one)

Reply:
I thought about it but I'm not sure it's worth the effort, as we either need to maintain the list manually or generate some kind of JSON file.

Keep in mind that when we refactor the navbar, we'll be using CSS-in-JS, so the stylelint config (for the appkit repo) is not needed anymore.